src/lib/components/Grid/__tests__/Grid.test.jsx

```diff
@@ -18,6 +18,16 @@ const responsiveBreakpoints = {
   x2l: 'placeholder-x2l',
   x3l: 'placeholder-x3l',
 };
+
+const responsiveSpacingBreakpoints = {
+  xs: 1,
+  sm: 2,
+  md: 3,
+  lg: 4,
+  xl: 5,
+  x2l: 6,
```

Comment:
I think we should use the more common convention `2xl` insteaed of `x2l`. In terms of ordering both solutions are equally non-elegant since we have `xs`as the first element anyway.

Reply:
> 2xl notation (starting with a digit) requires quotes which makes breakpoints syntax inconsistent.

See #354 where the naming changed.

Reply:
Ok then I guess. :(